src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -326,52 +329,52 @@ protected static async Task<ImmutableArray<SyntaxToken>> GetIdentifierOrGlobalNa
             return null;
         }
 
-        protected static Task<ImmutableArray<FinderLocation>> FindAliasReferencesAsync(
-            ImmutableArray<FinderLocation> nonAliasReferences,
+        protected static Task<ImmutableArray<FinderLocation>> FindLocalAliasReferencesAsync(
+            ArrayBuilder<FinderLocation> initialReferences,
```

Comment:
i stopped calling these 'nonAliasReferences' because these could be *global* aliases.  I also renamed a lot of 'FindAliasXXX' helpers to 'FindLocalAliases' to help keep the split between global and local aliases clearer.